docs/using-the-jdbc-driver/HostAvailabilityStrategy.md

```diff
@@ -0,0 +1,20 @@
+# Host Availability Strategy
```

Comment:
Can this page be added [here](https://github.com/awslabs/aws-advanced-jdbc-wrapper/blob/main/docs/Documentation.md)?

Reply:
Fixed!